pkg/monitoring/rules/rules.go

```diff
@@ -1,180 +1,54 @@
 )
 
 const (
-	severityAlertLabelKey     = "severity"
-	healthImpactAlertLabelKey = "operator_health_impact"
-	partOfAlertLabelKey       = "kubernetes_operator_part_of"
-	partOfAlertLabelValue     = "kubevirt"
-	componentAlertLabelKey    = "kubernetes_operator_component"
-	componentAlertLabelValue  = "ssp-operator"
+	ruleName             = "prometheus-k8s-rules-cnv"
+	PrometheusLabelKey   = "prometheus.ssp.kubevirt.io"
+	PrometheusLabelValue = "true"
 )
 
-const (
-	CommonTemplatesRestoredIncreaseQuery   = "sum(increase(kubevirt_ssp_common_templates_restored_total{pod=~'ssp-operator.*'}[1h]))"
-	TemplateValidatorRejectedIncreaseQuery = "sum(increase(kubevirt_ssp_template_validator_rejected_total{pod=~'virt-template-validator.*'}[1h]))"
-)
+func SetupRules() error {
+	err := recordingrules.Register()
+	if err != nil {
+		return err
+	}
 
-// RecordRulesDesc represent SSP Operator Prometheus Record Rules
-type RecordRulesDesc struct {
-	Name        string
-	Expr        intstr.IntOrString
-	Description string
-	Type        string
-}
+	err = alerts.Register()
+	if err != nil {
+		return err
+	}
 
-// recordRulesDescList lists all SSP Operator Prometheus Record Rules
-var recordRulesDescList = []RecordRulesDesc{
-	{
-		Name:        "kubevirt_ssp_operator_up",
-		Expr:        intstr.FromString("sum(up{pod=~'ssp-operator.*'}) OR on() vector(0)"),
-		Description: "The total number of running ssp-operator pods",
-		Type:        "Gauge",
-	},
-	{
-		Name:        "kubevirt_ssp_template_validator_up",
-		Expr:        intstr.FromString("sum(up{pod=~'virt-template-validator.*'}) OR on() vector(0)"),
-		Description: "The total number of running virt-template-validator pods",
-		Type:        "Gauge",
-	},
-	{
-		Name:        "kubevirt_ssp_operator_reconcile_succeeded_aggregated",
-		Expr:        intstr.FromString("sum(kubevirt_ssp_operator_reconcile_succeeded)"),
-		Description: "The total number of ssp-operator pods reconciling with no errors",
-		Type:        "Gauge",
-	},
-	{
-		Name:        "kubevirt_ssp_template_validator_rejected_increase",
-		Expr:        intstr.FromString(TemplateValidatorRejectedIncreaseQuery + " OR on() vector(0)"),
-		Description: "The increase in the number of rejected template validators, over the last hour",
-		Type:        "Gauge",
-	},
-	{
-		Name:        "kubevirt_ssp_common_templates_restored_increase",
-		Expr:        intstr.FromString(CommonTemplatesRestoredIncreaseQuery + " OR on() vector(0)"),
-		Description: "The increase in the number of common templates restored by the operator back to their original state, over the last hour",
-		Type:        "Gauge",
-	},
+	return nil
 }
 
-func RecordRules() []promv1.Rule {
-	result := make([]promv1.Rule, 0, len(recordRulesDescList))
-	for _, rrd := range recordRulesDescList {
-		result = append(result, promv1.Rule{Record: rrd.Name, Expr: rrd.Expr})
+func BuildPrometheusRule(namespace string) (*promv1.PrometheusRule, error) {
+	rules, err := operatorrules.BuildPrometheusRule(
```

Comment:
```suggestion
	return operatorrules.BuildPrometheusRule(
```

anmd remove the error handling below

Reply:
right sorry you tols me that in hco pr as well... fixed